### PR TITLE
Replace yarn with npm, remove deprecations, fix cache, fix template path

### PR DESCRIPTION
**Pull request by Definitely-Not-Vlad**
Fixes multiple issues, as described in title.

Can be tested with `npm i -g @shoutem/cli@rc`. RC version does not contain changes from [#aff7b3f](https://github.com/shoutem/cli/pull/75/commits/aff7b3faacdd1bbbfe761249e92c4082eb2bd3fc) commit (Clean up user facing messages).

Cache no longer overrides new installation.
Deprecation warnings have been removed as the commands will not be deprecated.
Templates are now fetched from `src` directory as they should not be transpiled via babel so they're ignored.
Fixed grammar and other details in some user facing messages.

I've also deleted the release/0.13.1 branch in favor of making a new release once this is merged, due to develop going too far ahead with 0.13.1 relevant commits instead of them being merged into release/0.13.1.

### CODE REVIEW

File: src/cli/extension/add.js [hunk 1]
```diff
@@ -66,8 +66,7 @@ export async function addExtension({ name, local, externalDestination }) {
 
   const cdCommand = 'cd ' + path.relative(process.cwd(), extensionPath);
   console.log('\nCongratulations, your new extension is ready!'.green.bold);
-  console.log(`You might try doing ${cdCommand.cyan} where you can:`);
+  console.log(`You can now do '${cdCommand.cyan}' where you can:`);
```
[the remark this answers is not in the document]
Comment (Definitely-Not-Vlad) [reply src/cli/extension/add.js:69]: It doesn't make sense that they would try doing the three things they need to do in order to make the extension anything other than "Hello world", hence the change to "You can now do".

File: src/cli/extension/publish.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ import {updateExtension, getInstallation, installExtension} from "../../clients/
 import confirmer from "../../services/confirmer";
 import {getExtension} from "../../clients/extension-manager";
 
-export const description = 'Publish an extension from the app in the working directory';
+export const description = 'Publish an extension from the app in the specified orcurrent working directory';
```
[the remark this answers is not in the document]
Comment (Definitely-Not-Vlad) [reply src/cli/extension/publish.js:12]: Thanks for spotting that!

File: src/cli/theme.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-export const description = 'Manage application themes';
+export const description = 'Manage extension themes';
```
Comment (toroz) [inline src/cli/theme.js:1]: Manages extension themes.

File: src/cli/init.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ import {executeAndHandleError} from '../services/error-handler';
 import {addExtension} from "./extension/add";
 
 export const command = 'init <name>';
-export const description = 'Create a scaffold of all files and folders required to build an extension.';
+export const description = 'Create boilerplate with all the files and folders required to build an extension';
```
Comment (toroz) [inline src/cli/init.js:5]: Creates a basic extension structure, with all the required files and folders.

File: src/cli/extension/publish.js [hunk 5]
```diff
@@ -9,7 +9,7 @@ import {updateExtension, getInstallation, installExtension} from "../../clients/
 import confirmer from "../../services/confirmer";
 import {getExtension} from "../../clients/extension-manager";
 
-export const description = 'Publish an extension from the app in the working directory';
+export const description = 'Publish an extension from the app in the specified or current working directory';
```
Comment (toroz) [inline src/cli/extension/publish.js:12]: . at the end

**Pull request merged**